README.md

```diff
@@ -588,6 +588,12 @@ var signIn = new OktaSignIn(config);
     helpSupportNumber: '(123) 456-7890'
     ```
 
+- **signOutUrl:** url to redirect when a user clicks the sign out link. If not provided, the widget will navigate to primary auth.
```

Comment:
1, 2 - will do
3 - I think I can create a section for Links and a sections for Buttons. It seems to me that so far we don't need to mixed them together

click function: I'm 50/50 in this one. So far links seems to be overriden with URLs, and buttons seem to have the click function. I think we can keep that pattern?